microsoft/botframework-solutions
# [Virtual Assistant Template][TypeScript] Add README for generated assistant, and update the description prompt

## Description

- Add a `README` for the generated assistant
- Update **description** prompt
- Add template `appsettings.json` file
- Delete unnecessary `README`
- Fix typos

Related to #1058 

## Testing Steps
1. Go to `AI\templates\Virtual-Assistant-Template\src\typescript\generator-botbuilder-assistant`
2. Run `npm link` to symlink the package folder
3. Open a terminal wherever you want to create an assistant
4. Run yo `botbuilder-assistant`
5. Follow the generator's prompts
6. Verify the new description prompt
7. Check that the generator finished successfully
8. Open the **generated assistant**
9. Verify that contains a `README` with the next steps and follow them

## Checklist
